### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by Jasonhyw** [Zhendong-Wang/Prompt-Diffusion#8]
Hi, how do I retrain using train.py? I would like to reproduce the experimental results first. I followed the scrpits/.sh file for training.
![f2e87a95ba27167c67569769cb65ea2](https://github.com/Zhendong-Wang/Prompt-Diffusion/assets/51027790/7b9a8364-491f-4a98-86e3-2a334e79c7d7)
Does the above code need to be commented?

**Comment by Zhendong-Wang** [comment]
No. The resume path is for resuming from the pretrained Stable Diffusion v1.5 checkpoint. Our model is finetuned based on that ckpt. Note you need to first run python tool_add_control.py to create the control_sd15_ini.ckpt.

**Comment by Jasonhyw** [comment]
What does the a_prompt and n_prompt mean? What are they used for？
